### PR TITLE
Feature/security profiles

### PR DESCRIPTION
**Pull request by southbite**
this is a big release, will need quite a bit of review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -435,6 +435,122 @@ happn.client.create({config:{username:'_ADMIN', password:'testPWD'}, secure:true
...
+            {
+              name:"web-session",
+              session:{
+                $and:[{
+                  user:{username:{$eq:'WEB_SESSION'}},
+                  type:{$eq:0}
+                }]
+              },
+              policy:{
+                ttl: 4000,
```
Comment (JensEggers) [inline README.md]: is this seconds or milliseconds? prob best if field name tells us

File: test/1_eventemitter_embedded_sanity.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 describe('1_eventemitter_embedded_sanity', function () {
 
-  require('benchmarket').start();
-  after(require('benchmarket').store());
+  // require('benchmarket').start();
```
Comment (JensEggers) [inline test/1_eventemitter_embedded_sanity.js]: why's this off?

**Comment by JohanvdWest** [comment]
Can we close this PR as #116 replaces it?

**Comment by JensEggers** [comment]
fine by me, closing

**Pull request closed without merging**